### ISSUE TITLE
Accessibility | Breadcrumb - separators aren't hidden in accessibility tree

### ISSUE THREAD

**Issue by fgeierst** [primefaces/primeng#19520]
### Describe the bug

While the list semantic is correct, the separators (chevron icon) should not be exposed as listitems. Also the chevron icons should not be exposed in the accessibility tree.

This is a duplicate if #13437, which is closed but partly not fixed.

<img width="2417" height="1440" alt="Image" src="https://github.com/user-attachments/assets/05983af9-3b49-4870-816d-7f939414987d" />

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-ynsptzwi?file=src%2Fapp%2Fapp.component.html

### Environment

/

### Angular version

21.2.4

### PrimeNG version

v21

### Node version

24.13.0

### Browser(s)

Chrome

### Steps to reproduce the behavior

1.  Open the https://primeng.org/breadcrumb page
2. Inspect the first example for the Breadcrumb component
3. The lis used as separators show up in the the accessibility tree

### Expected behavior

The lis used as separators DO NOT show in accessibility tree.

**Comment by fgeierst** [comment]
While this is a duplicate, the original ticket is closed but the issue is not resolved.
